src/components/ProblemsTable.tsx

```diff
@@ -281,30 +286,15 @@ export const ProblemsTable = ({
           minWidth: 50,
           maxWidth: 170,
           Cell: (row: CellProps<Report>) => {
-            const [errorId, groupCount] = row.row.original.issueid
-              ? row.row.original.issueid.split(' ', 2)
-              : [undefined, undefined];
+            const [errorId] = row.row.original.issueid ? row.row.original.issueid.split(' ', 2) : [undefined];
             return (
               <div>
                 {(row.row.subRows.length === 0 &&
                   context?.currentTable &&
                   tableStyleAccessor[context?.currentTable] === tableStyleAccessor.issueId) ||
-                !errorId ? (
```

Comment:
We need the errorId links to show up for a percentage of users so this should not be fully removed. Its visibility should be based on the if feature flags is true or false.

Reply:
added proper toggling condition